src/JSON For mIRC.mrc

```diff
@@ -848,7 +848,7 @@ alias JSON {
       if ($0 < 2) {
         %Error = INVALID_PARAMETERS
       }
-      elseif (!$len($2) || $isfile($2) || (!$regex($2, /[\\\/]/) && " isin $2)) {
+      elseif (!$len($2)) || ($isfile($2)) || (!$regex($2, /[\\\/]/)) && (" isin $2) {
```

Comment:
This is incorrect. The correct version would be the following but I choose not to include the extra `()`:
```
elseif (!$len($2)) || ($isfile($2)) || ((!$regex($2, /[\\\/]/)) && (" isin $2)) {
```